homeassistant/components/sensor/transmission.py

```diff
@@ -116,6 +122,50 @@ def update(self):
         self._transmission_api.update()
         self._data = self._transmission_api.data
 
+        if self.first_run:
```

Comment:
You seem to save a list of completed_torrents and started_torrents in all sensor instances, but it's only used by 1 sensor.

Is it not maybe better to save these variables in the TransmissionData class then you only have 1 copy per Transmission server?

Reply:
Nice suggestions, I get you point. I didn't think about the fact that for each variable a new sensor is created. I'll try to implement it asap.